pages/soundtrack/soundtrack.go

```diff
@@ -19,7 +19,16 @@ func Get(ctx aero.Context) error {
 		return ctx.Error(http.StatusNotFound, "Track not found", err)
 	}
 
+	relatedTracks := make([]*arn.SoundTrack, 0, 5)
+	for _, anime := range track.Anime() {
+		anime := anime
+		tracks := arn.FilterSoundTracks(func(t *arn.SoundTrack) bool {
```

Comment:
Having the database reading function `FilterSoundTracks` inside a loop can potentially be costly.

Just to be sure that this doesn't generate a lot of database requests, I'd like to do a **single** `FilterSoundTracks` call in the future.

Reply:
Ok, I've fixed it. Sorry, I kinda overlooked that.